docs/js-core-3/week-2/lesson.md

```diff
@@ -399,40 +399,92 @@ fetch(_____)
 
 :::
 
-## PD (Agile: Planning and preparation)
+## PD (How to give and receive feedback effectively)
 
-**Session objective:** In this session we will talk about refining a user story backlog, planning the sprint and estimating effort.
+**Session objective:** In this session we will look into different ways of providing feedback.
 
-:::note Warm-up (10 minutes)
+Feedback is a valuable tool for indicating whether things are going in the right direction or whether re-direction is required. It is also an effective way to help people develop and grow professionally. How feedback is communicated will not only determine how effective it will be but also how positive its impact will be. In this session, we will look into different ways of providing feedback and will go through different scenarios that will help us understand the power of providing and receiving effective feedback.
 
-Write a mission statement for the TV show project:
+:::note Exercise (5 min)
 
-- What's the purpose of the project?
-- What would be the mission of the sprints?
+**Exercise objective:** To understand positive and negative feedback.
+
+Let’s share our understanding of what feedback is. Volunteer to explain your definition of feedback.
+
+Once a few trainees have expressed their opinions, volunteer to describe the differences between positive and negative feedback. Do you think negative feedback is a bad thing?
 
 :::
 
-[Agile: Planning and preparation](https://docs.google.com/presentation/d/1TJNjPOha2P5cWW6IO9jAc9GTvsosqVGkkG5pAbbvfg4/edit#slide=id.g1b740a9f8c6_0_336)
+**Definitions**
+
+**Positive feedback**: This is the type of feedback that we all want to hear, it’s when someone praises our work.
+
+Appreciation and positive remarks in the workplace can help an employee feel appreciated, can help to reinforce good behaviours and will help employees with their professional development.
+However, it is important not to overuse positive feedback as its value will decrease. You may find it more valuable to hear positive comments from someone who almost never praises you for your work than a coworker constantly telling you how awesome you are. Too much positive feedback can also make employees complacent and feel less challenged in their roles.
```

Comment:
I realise this isn't an exhaustive guide, but I'd think about including something about how _specific_ positive feedback can be much more effective than general positive feedback.

"They way you presented graphs to show our progress really made it easy to notice what tasks I needed to focus on which helped get the project delivered" vs "It was great working with you, it helped the project succeed"

Reply:
I like this!